### ISSUE TITLE
No such file or directory @ dir_chdir after upgrading to 0.42.0

### ISSUE THREAD

**Issue by ianmurrays** [castwide/solargraph#445]
Hi, after upgrading to 0.42.0, I'm now getting this error when running `solargraph scan -v`:

```
solargraph-0.42.0/lib/solargraph/documentor.rb:62:in `chdir': No such file or directory @ dir_chdir -  (Errno::ENOENT)
```

Using ruby 2.5.8.

**Comment by IvanBernatovic** [comment]
Same issue in Rails app on Ruby 2.6.5. Using VS code Solargraph extension.

**Comment by castwide** [comment]
Patched in version 0.42.1.
